charts/publisher/templates/web/dbmigration-job.yaml

```diff
@@ -1,8 +1,12 @@
 {{- if .Values.dbMigrationEnabled -}}
-{{/* Job objects are immutable but Helm still tries to patch them when they
-change, so we suffix the name with the chart version and image tag to create a
-new one when the chart or the binary changes. */}}
-{{- $jobName := print .Release.Name "-dbmigrate-" .Chart.Version "-" .Values.image.tag -}}
+{{/*
+k8s Jobs are immutable but Helm still tries to patch them when they change, so
+we suffix the name with the chart version and image tag to create a new one
+when the chart or the binary changes. This workaround doesn't work when moving
+a mutable tag (like "latest"), but we shouldn't be using those anyway.
```

Comment:
Yeah that's a good point. Once we have the GitOps release automation stuff (i.e. once we have Argo updating the image label in the spec in `argocd-apps/values-$env.yaml`) then it'd be worth considering enforcing immutability 👍

Reply:
I dunno how much of a pain it'd be for development if we didn't have `latest` though, but definitely worth considering. (some discussion in https://github.com/aws/containers-roadmap/issues/ issue 878 [not linked directly to avoid refspam])